libbeat/publisher/queue/queue.go

```diff
@@ -46,6 +46,9 @@ type Metrics struct {
 	//EventLimit is the user-configured event limit of the queue
 	EventLimit opt.Uint
 
+	//OccupiedRead is the count of events that an output consumer has read, but not yet ack'ed
+	OccupiedRead opt.Uint
```

Comment:
Could we call this something more self-explanatory? Like just shortening the comment into the variable name `EventsReadButNotYetAcked`? Maybe it could be shortened to `UnackedConsumedEvents` or something.

Reply:
Just gonna do `UnackedConsumedEvents` unless @faec has other ideas.